src/managing-workflow/platform-monitoring.md

```diff
@@ -1,3 +1,58 @@
 # Platform Monitoring
 
-Platform monitoring is a work in progress. If you wish to follow the progress you can do so [here](https://github.com/deis/monitor).
\ No newline at end of file
+## Description
+With the release of Workflow Beta4 we now include a monitoring stack for introspection on a running Kubernetes cluster. The stack includes 4 components:
+* [Telegraf](https://docs.influxdata.com/telegraf/v0.12/) - Metrics collection daemon written by team behind InfluxDB.
+* [InfluxDB](https://docs.influxdata.com/influxdb/v0.12/) - Time series database
+* [Grafana](http://grafana.org/) - Graphing tool for time series data
+* [Stdout-Metrics](https://github.com/deis/stdout-metrics) - Tool for consuming metrics via standard out and forwards them to InfluxDB
+
+## Architecture Diagram
+
+```
+┌────────┐                  
+│ Router │                  
+└────────┘                  
+     │                      
+     │                      
+     ▼          ┌──────────┐
+┌─────────────┐         ┌─────────┐     │ stdout   │
+│ HOST        │         │ fluentd │────▶│  metrics │
+│  Telegraf   │───┐     └─────────┘     └──────────┘
+└─────────────┘   │                           │     
+│                           │     
+┌─────────────┐   │                           │     
+│ HOST        │   │    ┌───────────┐          │     
+│  Telegraf   │───┼───▶│ InfluxDB  │◀─────────┘     
+└─────────────┘   │    └───────────┘                
+│          ▲                      
+┌─────────────┐   │          │                      
+│ HOST        │   │          │                      
+│  Telegraf   │───┘    ┌──────────┐                 
+└─────────────┘        │ Grafana  │                 
+└──────────┘                 
+```
+
+### Grafana
+We expose Grafana through the router using service annotations. This allows users to access the Grafana UI by accessing `grafana.mydomain.com`. While  we provide a default username/password of `admin/admin` this can be overridden at any time by setting the following environment variables in `deis-monitor-grafana-rc.yaml`: `GRAFANA_USER` and `GRAFANA_PASSWD`.
```

Comment:
could you add a link here to docs on how the router works with service annotations?